### DISCUSSION TITLE
极空间NAS设备的8123端口无法使用

### DISCUSSION THREAD

**Discussion by UnderFFFox** [XiaoMi/ha_xiaomi_home#116] [Ideas]
极空间NAS设备的8123端口是默认被占用的，安装HA需要自定义端口，登录页面的地址不可修改的话，极空间NAS不可用

**Comment by JimmyKmi** [comment]
参考 https://note.p.lzj.ac.cn/s/bd635048-ae38-43c9-9f31-d61175a01cf6/doc/55m75b2v6zug5oiq5asx6lsl6zeu6aky-WvtZZGylQy

**Comment by davidyamado** [comment]
@UnderFFFox 你这个问题解决了吗？

**Reply by UnderFFFox** [reply]
源码里的地址和端口我修改过，地址栏里地址也手动改过，都没用，等再更新几版看看吧，最近没折腾

**Reply by davidyamado** [reply]
我修改之后可以进去，可以读取到设备，但是无法获取设备状态

**Comment by NXY666** [comment]
登录完成后它会跳转到一个url为“http://homeassistant.local:8123/xxxx ”的页面。

等它报错后，把协议、域名和端口改成能进你家ha的url。

比如说我会把它改成“http://192.168.1.114:8123/xxxx ”，然后再回车访问，就能完成登录了。

**Reply by UnderFFFox** [reply]
试过，还是不行，你说的地址会转成16位的，我手动改过，源码里也改过这个认证地址，都不行，所以放弃了，再等几个版本试试

**Reply by NXY666** [reply]
> 试过，还是不行，你说的地址会转成16位的，我手动改过，源码里也改过这个认证地址，都不行，所以放弃了，再等几个版本试试

如果是的话电脑还能改hosts，把这个.local地址直接链接到你的服务器IP。

**Reply by UnderFFFox** [reply]
改hosts没试，抽空折腾下
